Sources/Graphics/Gif/Image.php

```diff
@@ -179,9 +179,9 @@ public function deInterlace(): void
 				$this->m_data = substr($this->m_data, $this->m_gih->m_nWidth);
 
 				$data =
-					substr($data, 0, $y * $this->m_gih->m_nWidth) .
+					substr($data, 0, (int) ($y * $this->m_gih->m_nWidth)) .
 					$lne .
-					substr($data, ($y + 1) * $this->m_gih->m_nWidth);
+					substr($data, (int) (($y + 1) * $this->m_gih->m_nWidth));
```

Comment:
Casting the result of the expression to int is unnecessary. The result of the arithmetic operation will already be an integer.